cabal.project

```diff
@@ -107,6 +107,18 @@ package cardano-ledger-alonzo-test
 -- --------------------------- 8< --------------------------
 -- Please do not put any `source-repository-package` clause above this line.
 
+source-repository-package
```

Comment:
Can we leave a comment that we want to eventually not rely on our own fork after the relevant PR has been merged?